examples/axi_lite_slave/tests/test_axi_lite_slave.py

```diff
@@ -165,7 +165,7 @@ def write_fail(dut):
         yield axim.write(ADDRESS, DATA)
         yield Timer(CLK_PERIOD * 10)
     except AXIProtocolError as e:
-        print("Exception: %s" % str(e))
+        print ("Exception: %s" % str(e))
```

Comment:
`print (` is not typical syntax in this repo, could you change it back to `print(` (or was this triggered by another comment which I've missed?)?

Reply:
I'll change it back. I think there might have been a conflict somewhere along the lines and I chose poorly in the rebase.